### PR TITLE
fix(route): add openai research support

### PR DESCRIPTION
**Pull request by yuguorui**
## 该 PR 相关 Issue / Involved Issue

Close #

## 路由地址示例 / Example for the Proposed Route(s)

```routes
/openai/research
```

## 新 RSS 路由检查表 / New RSS Route Checklist
  
- [x] 新的路由 New Route
  - [x] 跟随 [v2 路由规范](https://docs.rsshub.app/joinus/script-standard.html) Follows [v2 Script Standard](https://docs.rsshub.app/en/joinus/script-standard.html)
- [x] 文档说明 Documentation
  - [x] 中文文档 CN
  - [x] 英文文档 EN
- [x] 全文获取 fulltext
  - [x] 使用缓存 Use Cache
- [ ] 反爬/频率限制 anti-bot or rate limit?
  - [ ] 如果有, 是否有对应的措施? If yes, do your code reflect this sign?
- [x] [日期和时间](https://docs.rsshub.app/joinus/pub-date.html) [date and time](https://docs.rsshub.app/en/joinus/pub-date.html)
  - [x] 可以解析 Parsed
  - [x] 时区调整 Correct TimeZone
- [ ] 添加了新的包 New package added
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/v2/openai/research.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    const obj = resp.data;
+
+    const items = await Promise.all(
+        obj.data.map((item) => {
+            const attributes = item.attributes;
+            return parseArticle(ctx, researchRootUrl, attributes);
+        })
+    );
+
+    const title = `OpenAI Research`;
```
Comment (TonyRL) [inline lib/v2/openai/research.js:25]:
```suggestion
    const title = 'OpenAI Research';
```

File: docs/new-media.md [hunk 2]
```diff
@@ -1040,6 +1040,10 @@ IPFS 网关有可能失效，那时候换成其他网关。
 
 <Route author="ETiV" example="/openai/chatgpt/release-notes" path="/openai/chatgpt/release-notes" />
 
+### Research
+
+<RouteEn author="yuguorui" example="/openai/research" path="/openai/research" />
```
Comment (TonyRL) [inline docs/new-media.md:1045]:
```suggestion
<Route author="yuguorui" example="/openai/research" path="/openai/research" />
```

File: lib/v2/openai/common.js [hunk 3]
```diff
@@ -0,0 +1,72 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+const getApiUrl = async () => {
```
Comment (TonyRL) [inline lib/v2/openai/common.js:6]: Double slash in https://github.com/DIYgod/RSSHub/actions/runs/4938783244/jobs/8828910028#step:12:17

This affect both `/blog/:tag?` and `/research`

Comment (yuguorui) [reply lib/v2/openai/common.js:6]: Yes, you are right. Let's fix this by using standard library function `new URL(input[, base])`.

**Pull request merged**